swarm/network/simulation/node.go

```diff
@@ -326,3 +340,16 @@ func (s *Simulation) StopRandomNodes(count int) (ids []enode.ID, err error) {
 func init() {
 	rand.Seed(time.Now().UnixNano())
 }
+
+// derive a private key for swarm for the node key
+// returns the private key used to generate the bzz key AND the generated bzz key
+func BzzKeyFromConfig(conf *adapters.NodeConfig) (*ecdsa.PrivateKey, []byte, error) {
+	// ecdsa.GenerateKey takes 40 bytes entropy
+	privKeyBuf := append(crypto.FromECDSA(conf.PrivateKey), []byte{0x62, 0x7a, 0x7a, 0x62, 0x7a, 0x7a, 0x62, 0x7a}...)
```

Comment:
Could you extract `[]byte{0x62, 0x7a, 0x7a, 0x62, 0x7a, 0x7a, 0x62, 0x7a}` into a variable so that we don't have magic constants? What these hex number anyway? I don't know what they mean.

Reply:
Hint: hex.

And I tend to think that a bit of cryptic easter egg fun in the code raises spirits-

Reply:
I see these are hex numbers, but still **magic constants**. <- :fire:

So, I guess I disagree.
As I feel we already have enough cryptic code to understand. :wink:
Let's make fun of solving the already existing puzzles first. :+1:

Reply:
> Let's make fun of solving the already existing puzzles first. 

That _would_ be magic.

Reply:
fun is ok bzz, but why are we doing this again? what  was wrong with this?